test/test_packages/monorepo/packages/C/Project.toml

```diff
@@ -0,0 +1,8 @@
+name = "C"
+uuid = "4ee78ca3-4e78-462f-a078-747ed543fa86"
+version = "0.0.0"
+parent = "A"
+
+[deps]
+D = "bf733257-898a-45a0-b2f2-c1c188bdd870"
```

Comment:
Yeah, this allows to avoid having to duplicate the paths over and over for all subpackages, but instead the parent `[sources]` is like a mini path registry that subpackages can reference. It makes subpackage relocation much easier.